lisa/sut_orchestrator/azure/arm_template.json

```diff
@@ -42,6 +48,7 @@
         "adminUserName": "[parameters('adminUserName')]",
         "adminPassword": "[parameters('adminPassword')]",
         "adminKeyData": "[parameters('adminKeyData')]",
+        "availabilitySetTags": "[parameters('availabilitySetTags')]",
```

Comment:
Is there a way we could validate these JSON files to catch typos and whatnot?  Is there a schema we could validate them against, especially one that could validate these values?

Reply:
Yes, there is an vscode extension called `Azure Resource Manager (ARM) Tools`, it can validate more than schema. For example, warn on unused variables, unmatched []/''. There is another powershell script for "best practice", it do more than the extension, but a lot of unfixable warnings, so that I cannot enable it in pipeline.

Reply:
Can the code from that vscode extension be called outside vscode and added to the CI? Not everybody uses IDEs, or will have that extension set up properly.  (This is of course unrelated to this PR. Just thinking out loud here.)

Reply:
It looks there are some tools. added to backlog.